src/lib/jsClient.ts

```diff
@@ -29,25 +29,21 @@ import { resolveRelay } from "./multiaddres.js";
 
 export const initFluenceClient = async (
   {
-    relay: maybeRelay,
+    relay: relayFromFlags,
     sk: secretKeyName,
     ["dial-timeout"]: dialTimeoutMs,
     ttl,
     "particle-id": printParticleId,
+    [ENV_FLAG_NAME]: fluenceEnvFromFlags,
   }: FluenceClientFlags & KeyPairFlag,
   maybeFluenceConfig: FluenceConfig | null,
-  fluenceEnv: FluenceEnv,
 ): Promise<void> => {
   const relay = await resolveRelay({
-    fluenceEnv,
     maybeFluenceConfig,
-    maybeRelay,
+    relayFromFlags,
+    fluenceEnvFromFlags,
   });
 
-  commandObj.logToStderr(
-    `Connecting to ${color.yellow(fluenceEnv)} relay: ${relay}`,
```

Comment:
I believe this log is helpful to know what is happening. Could we leave it here and not print env if it is unknown or something like that?